src/bulls_and_cows_lib/game.cpp

```diff
@@ -1,20 +1,21 @@
-
 #include "game.hpp"
 #include "board.hpp"
 #include "game_options.hpp"
 #include "game_solver.hpp"
 #include "input.hpp"
 #include "main_menu.hpp"
```

Comment:
Sais-tu comment corriger ces warnings de link ?
[build] game.cpp.obj : warning LNK4006: "int __cdecl bulls_and_cows::generate_random_integer(int,int)" (?generate_random_integer@bulls_and_cows@@YAHHH@Z) already defined in random.cpp.obj; second definition ignored
[build] game.cpp.obj : warning LNK4006: "char __cdecl bulls_and_cows::generate_random_character(char,char)" (?generate_random_character@bulls_and_cows@@YADDD@Z) already defined in random.cpp.obj; second definition ignored

Reply:
oui j'ai réglé le problème normalement

Reply:
ok